ContentFiles/Forms/FormsUIContent/App.xaml.cs.pp

```diff
@@ -0,0 +1,12 @@
+﻿using Xamarin.Forms;
+
+namespace $rootnamespace$
```

Comment:
I think this namespace should be `FormsUI`, as that is assumed later when referencing this `App` class

Reply:
Ok, so this reply will cover all of the comments around the namespaces. Everything I've done is based on the assumption that $rootnamespace$ is the project name, not the solution name. So in this case '$rootnamespace$' should be replaced by the user with 'YourSolutionName.FormsUI' because that's the project name (meaning you would went up with 'YourSolutionName.FormsUI.App'). I think that by default when you create a new project the namespace for that project is the project name, hence why I went with that. I can change this to the solution name if you want, but I'd have to change every namespace in this pull request to make that consistent.

This brings up another question: Are these Content Files still automatically used by NuGet to insert into people's projects when they install any of the MvvmCross NuGet packages? I didn't see this happen with the main packages, but haven't poked around further to see if they're other projects. I kind of assume they don't currently get automatically copied and so are more just sample files people can reference, but might get automatically copied in the future.